### ISSUE TITLE
Move View Parser Log and Ride Log from Tools to View toolbar menu

### ISSUE THREAD

**Issue by Nyral** [robotframework/RIDE#2254]
We should add all the menu items for viewing plugins to the View toolbar menu in order to maintain consistency.

What do you think? @HelioGuilherme66

**Comment by HelioGuilherme66** [comment]
I agree. Then we release some options from Tools to a more empty menu View.

**Comment by HelioGuilherme66** [comment]
Moving to 2.1 because of lack of resources (time and people)
